meganav.yaml

```
@@ -1,3 +1,699 @@
+              url: https://help.ubuntu.com/community/InternetRelayChat
+      secondary_links:
+        - title: Links
+          links:
+            - title: "Forum: New to Ubuntu"
+              url: https://ubuntuforums.org/forumdisplay.php?f=125
+            - title: "Forum: Ubuntu Official Flavours Support"
+              url: https://ubuntuforums.org/forumdisplay.php?f=327
+            - title: "Forum: Ubuntu, Linux and OS Chat"
+              url: https://ubuntuforums.org/forumdisplay.php?f=434
+            - title: Discourse Community category
+              url: https://discourse.ubuntu.com/c/community/78
+    - title: Contribute to Ubuntu
+      primary_links:
+        - title: Resources
+          links:
+            - title: Translate Ubuntu
+              description: Translate into your language
+              url: https://discourse.ubuntu.com/t/translations/32
+            - title: QA
+              description: Test releases and features
+              url: https://discourse.ubuntu.com/c/quality/10
+            - title: Report Bugs
+              description: Report bugs on Launchpad
+              url: https://help.ubuntu.com/community/ReportingBugs
+            - title: Improve documentation
+              description: Help write the docs
+              url: https://wiki.ubuntu.com/DocumentationTeam/SystemDocumentation
+            - title: Local community teams
+              description: Join a community team
+              url: http://loco.ubuntu.com/teams/
+            - title: Donate
+              description: Donate to the Ubuntu project
+              url: /download/desktop/thank-you
+    - title: Mission
+      primary_links:
+        - title: Resources
+          links:
+            - title: Our mission
+              description: What we stand for
+              url: /community/mission
+            - title: Community council
+              description: Governing body for the project
+              url: https://discourse.ubuntu.com/t/what-is-the-ubuntu-community-council/706
+            - title: Ubuntu code of conduct
+              description: Expectations of participants
+              url: /community/code-of-conduct
+            - title: Diversity policy
+              description: Our stance on Diversity
+              url: /community/diversity
+            - title: Project governace
+              description: How the project is governed
+              url: /community/governance
+      secondary_links:
+        - title: Links
+          links:
+            - title: Community Council Agenda
+              url: https://discourse.ubuntu.com/t/community-council-meeting-agendas-schedule/708
+            - title: Ubuntu Technical Board
+              url: https://wiki.ubuntu.com/TechnicalBoard
+            - title: Technical Board Agenda
+              url: https://wiki.ubuntu.com/TechnicalBoardAgenda
+    - title: Keep up to date
+      primary_links:
+        - title: Resources
```

```suggestion
        - title: Keep up to date
```
I am guessing these are all purposely set as 'Resources', but will highlight just in case